index.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,600;0,700;1,400&display=swap" rel="stylesheet"
+/>
+    <link href="Setting.css" rel="stylesheet"/>
+    <title>Survey Form</title>
+   </head>
+    <body>
+<div class="container">
+    <header class="text-center">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">
+            <em>Thank you for taking the time to help us improve the platform</em>
+        </p>
+    </header>
+
+    <form action="#" id="survey-form" class="form-label">
+        <div class="form-group">
+            <label for="Name" id="Name-label">Name</label>
+            <input 
+            type="text" 
+            id="Name" 
+            classe="form-control"
+            placeholder="Enter Your Name" 
+            required
+            />
+
+        </div>
+
+        <div class="form-group">
+            <label for="Email" id="Email-label">Email</label>
+            <input 
+              type="text" 
+              id="Email" 
+              classe="form-control" 
+              placeholder="Enter Your Email" 
+              required
+            />
+
+        </div>
+
+        <div class="form-group">
+            <label for="number" id="number-label">Age (optional)</label>
+            <input 
+            type="number" 
+            name="age"
+            id="number" 
+            min="10"
+            max="99"
+            classe="form-control" 
+            placeholder="Age" 
+            required
+            />
+
+        </div>
+
+        <div class="form-group">
+            <label for="dropdown">
+                Which option best describes your current role?
+            </label>
+            <select name="dropdown" id="dropdown" classe="form-control"required>
+            <option disabled>select current role</option>
+            <option value="Student">Student</option>
+            <option value="Full Time Job">Full Time Job</option>
+            <option value="Full Time Learner">Full Time Learner</option>
+            <option value="Prefer not to say">Prefer not to say</option>
+            <option value="Other">Other</option>
+            </select> 
```

Comment:
Good job so far!
Please kindly modify some issues that you still need to modify in the selection option from the dropdown list. one of the dropdown items lists is disabled. we shall change to  placeholder  "select current role" and modify the dropdown options  list item